apple/internal/apple_xcframework_import.bzl

```diff
@@ -299,6 +476,119 @@ def _apple_dynamic_xcframework_import_impl(ctx):
 
     return providers
 
+def _apple_static_xcframework_import_impl(ctx):
+    """Implementation of apple_static_xcframework_import."""
+    actions = ctx.actions
+    alwayslink = ctx.attr.alwayslink
+    apple_fragment = ctx.fragments.apple
+    apple_mac_toolchain_info = ctx.attr._mac_toolchain[AppleMacToolsToolchainInfo]
+    cc_toolchain = find_cpp_toolchain(ctx)
+    deps = ctx.attr.deps
+    disabled_features = ctx.disabled_features
+    features = ctx.features
+    grep_includes = ctx.file._grep_includes
+    has_swift = ctx.attr.has_swift
+    label = ctx.label
+    linkopts = ctx.attr.linkopts
+    xcframework_imports = ctx.files.xcframework_imports
+    xcode_config = ctx.attr._xcode_config[apple_common.XcodeVersionConfig]
+
+    xcframework = _classify_xcframework_imports(ctx.var, xcframework_imports)
+    target_triplet = cc_toolchain_info_support.get_apple_clang_triplet(cc_toolchain)
+
+    # if xcframework.bundle_type == _BUNDLE_TYPE.frameworks:
```

Comment:
Is the thought that a comment here makes cherry-picking easier? If so, can we leave a comment to that effect?

Reply:
no i think we should toss this once we're sure it works